Doppler.MercadoPagoApi/Controllers/PaymentController.cs

```diff
@@ -1,15 +1,29 @@
+using Doppler.MercadoPagoApi.DopplerSecurity;
+using Doppler.MercadoPagoApi.Models;
+using Doppler.MercadoPagoApi.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
 
 namespace Doppler.MercadoPagoApi.Controllers
 {
     [Authorize]
     [ApiController]
     public class PaymentController : Controller
     {
-        public PaymentController()
+        private readonly IMercadoPagoService _mercadoPagoService;
+
+        public PaymentController(IMercadoPagoService mercadoPagoService)
         {
+            _mercadoPagoService = mercadoPagoService;
+        }
 
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPost("/customers/{customername}")]
+        public async Task<IActionResult> CreateCustomerAsync(CreateCustomerRequest request)
+        {
+            var customer = await _mercadoPagoService.CreateCustomerAsync(request.Customer);
```

Comment:
Why not simply receive the customer?

```suggestion
        public async Task<IActionResult> CreateCustomerAsync(CustomerDto customer)
        {
            var customer = await _mercadoPagoService.CreateCustomerAsync(customer);
```